### PR TITLE
Add R. B. Fisher/Univ of Edinburgh to csrankings-r.csv

### PR DESCRIPTION
**Pull request by rbfisher3**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [X] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-r.csv [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name,affiliation,homepage,scholarid
 R. A. Bodnar,University of Southampton,https://www.ecs.soton.ac.uk/people/rab1e14,NOSCHOLARPAGE
+R. B. Fisher,University of Edinburgh,https://homepages.inf.ed.ac.uk/rbf/,LigYduEAAAAJ
```
Comment (emeryberger) [inline csrankings-r.csv:3]: Names must correspond to DBLP names - these appear to be "Robert B. Fisher" and "Bob Fisher". Both need to be included, and "R. B. Fisher" should be removed.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by rbfisher3** [comment]
Hi Emery

Thanks for the email.

My entry:
   R. B. Fisher,University of Edinburgh,https://homepages.inf.ed.ac.uk/rbf/,LigYduEAAAAJ
should be
   Robert B. Fisher,University of Edinburgh,https://homepages.inf.ed.ac.uk/rbf/,LigYduEAAAAJ
as you inferred.

I forked a copy to make the change to: csrankings-r.csv
but it appears that the change has already been made and I have an entry there
alreaddy under the name: Robert B. Fisher.
So, I'm not sure if anything more needs to be done.

Also, when I expand University of Edinburgh on:
   https://csrankings.org/#/fromyear/2011/toyear/2021/index?vision&europe
I do not see myself listed there. Maybe this is just due to a lag in rebuilding
csrankings.csv ?

Please advise.

Many thanks, Bob




On Thu, 13 Jan 2022, Emery Berger wrote:

> Date: Thu, 13 Jan 2022 00:22:29
> From: Emery Berger ***@***.***>
> Reply-To: emeryberger/CSrankings
>     ***@***.***>
> To: emeryberger/CSrankings ***@***.***>
> Cc: rbfisher3 ***@***.***>, Author ***@***.***>
> Subject: Re: [emeryberger/CSrankings] Add R. B. Fisher/Univ of Edinburgh to
>     csrankings-r.csv (PR #4347)
> 
> This email was sent to you by someone outside the University. You should only click on links or attachments if you are certain that the email
> is genuine and the content is safe.
> 
> @emeryberger requested changes on this pull request.
> 
> ______________________________________________________________________________________________________________________________________________
> 
> In csrankings-r.csv:
> 
> > @@ -1,5 +1,6 @@
>  name,affiliation,homepage,scholarid
>  R. A. Bodnar,University of Southampton,https://www.ecs.soton.ac.uk/people/rab1e14,NOSCHOLARPAGE
> +R. B. Fisher,University of Edinburgh,https://homepages.inf.ed.ac.uk/rbf/,LigYduEAAAAJ
> 
> Names must correspond to DBLP names - these appear to be "Robert B. Fisher" and "Bob Fisher". Both need to be included, and "R. B. Fisher"
> should be removed.
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android.
> You are receiving this because you authored the
> thread.[ARWQCRLVTVK7LAIM2EG2I2TUVYLMLA5CNFSM5LHEQTXKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOGK5UOUI.gif] Message
> ID: ***@***.***>
> 
> 
>


================

Prof. Robert B. Fisher
School of Informatics, Univ. of Edinburgh
1.11 Bayes Centre, 47 Potterrow, Edinburgh EH8 9BT, UK
+44-(131)-651-3441 (direct line), +44-(131)-651-3443 (secretary)
plus code: WRW7+54 E-MAIL: r.b.fisher ***@***.*** ed.ac.uk
http://www.inf.ed.ac.uk/people/staff/Robert_Fisher.html


The University of Edinburgh is a charitable body, registered in
Scotland, with registration number SC005336.

**Comment by emeryberger** [comment]
Rebuilding is done roughly monthly, together with updates of the DBLP database.

All of your entries were in fact already in the database, so I will close the PR.

You do not have any papers in the indexed vision conferences in the time span you selected. Here are all of the indexed papers for you (from DBLP) after 2003.

```json
  {
    "name": "Robert B. Fisher",
    "year": 2018,
    "title": "Hybrid Multi-camera Visual Servoing to Moving Target.",
    "conf": "IROS",
    "area": "iros",
    "institution": "University of Edinburgh",
    "numauthors": 5,
    "volume": "0",
    "number": "0",
    "startPage": 1132,
    "pageCount": 6
  },
  {
    "name": "Robert B. Fisher",
    "year": 2020,
    "title": "Real-time Stereo Visual Servoing for Rose Pruning with Robotic Arm.",
    "conf": "ICRA",
    "area": "icra",
    "institution": "University of Edinburgh",
    "numauthors": 7,
    "volume": "0",
    "number": "0",
    "startPage": 7050,
    "pageCount": 7
  },
  {
    "name": "Robert B. Fisher",
    "year": 2020,
    "title": "Learning Object-Centric Representations of Multi-Object Scenes from Multiple Views.",
    "conf": "NeurIPS",
    "area": "nips",
    "institution": "University of Edinburgh",
    "numauthors": 3,
    "volume": "0",
    "number": "0",
    "startPage": -1,
    "pageCount": -1
  },
```

**Comment by rbfisher3** [comment]
Hi again. I'm still confused. My DBLP entry has 254 records, mostly computer vision
publications, with e.g. 9 pubs in 2021. A similar number are in my google scholar page.
I'm unclear why they are not included in the csranking list, even if I use dates 1980-2021.
At least 80 of my publications are journal pubs.

Does the csranking filter only select publications where I am the first-author?

Also, my entry seems to classify me as 'ROBOTICS', whereas most of my publications are 'VISION'.
Where is this label set?

Sorry to be a pest.

Many thanks, Bob


On Thu, 13 Jan 2022, Emery Berger wrote:

> Date: Thu, 13 Jan 2022 01:27:05
> From: Emery Berger ***@***.***>
> Reply-To: emeryberger/CSrankings
>     ***@***.***>
> To: emeryberger/CSrankings ***@***.***>
> Cc: rbfisher3 ***@***.***>, Author ***@***.***>
> Subject: Re: [emeryberger/CSrankings] Add R. B. Fisher/Univ of Edinburgh to
>     csrankings-r.csv (PR #4347)
> 
> This email was sent to you by someone outside the University. You should only click on links or attachments if you are certain that the email
> is genuine and the content is safe.
> 
> Rebuilding is done roughly monthly, together with updates of the DBLP database.
> 
> All of your entries were in fact already in the database, so I will close the PR.
> 
> You do not have any papers in the indexed vision conferences in the time span you selected. Here are all of the indexed papers for you (from
> DBLP) after 2003.
>
>   {
>     "name": "Robert B. Fisher",
>     "year": 2018,
>     "title": "Hybrid Multi-camera Visual Servoing to Moving Target.",
>     "conf": "IROS",
>     "area": "iros",
>     "institution": "University of Edinburgh",
>     "numauthors": 5,
>     "volume": "0",
>     "number": "0",
>     "startPage": 1132,
>     "pageCount": 6
>   },
>   {
>     "name": "Robert B. Fisher",
>     "year": 2020,
>     "title": "Real-time Stereo Visual Servoing for Rose Pruning with Robotic Arm.",
>     "conf": "ICRA",
>     "area": "icra",
>     "institution": "University of Edinburgh",
>     "numauthors": 7,
>     "volume": "0",
>     "number": "0",
>     "startPage": 7050,
>     "pageCount": 7
>   },
>   {
>     "name": "Robert B. Fisher",
>     "year": 2020,
>     "title": "Learning Object-Centric Representations of Multi-Object Scenes from Multiple Views.",
>     "conf": "NeurIPS",
>     "area": "nips",
>     "institution": "University of Edinburgh",
>     "numauthors": 3,
>     "volume": "0",
>     "number": "0",
>     "startPage": -1,
>     "pageCount": -1
>   },
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android.
> You are receiving this because you authored the
> thread.[ARWQCROZHTHJAC62NQXLKQDUVYS6TA5CNFSM5LHEQTXKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOHRGFIVA.gif] Message ID:
> ***@***.***>
> 
> 
>


================

Prof. Robert B. Fisher
School of Informatics, Univ. of Edinburgh
1.11 Bayes Centre, 47 Potterrow, Edinburgh EH8 9BT, UK
+44-(131)-651-3441 (direct line), +44-(131)-651-3443 (secretary)
plus code: WRW7+54 E-MAIL: r.b.fisher ***@***.*** ed.ac.uk
http://www.inf.ed.ac.uk/people/staff/Robert_Fisher.html


The University of Edinburgh is a charitable body, registered in
Scotland, with registration number SC005336.

**Pull request closed without merging**